src/item.cpp

```diff
@@ -3245,16 +3245,24 @@ void item::on_damage( int, damage_type )
 std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int truncate ) const
 {
     std::stringstream ret;
-
+    std::string dirt_symbol = to_string( ( int ) get_var( "dirt", 0 ) );
```

Comment:
This is a c-style cast, which we don't use.
```suggestion
    std::string dirt_symbol = to_string( static_cast<int>( get_var( "dirt", 0 ) ) );
```